pkg-builder.pl

```diff
@@ -186,32 +112,9 @@ ($)
       "--pkg-release=$pkg_info->{revision}",
       "--pkg-summary=$pkg_info->{summary}",
       "--pkg-post-install-script=scripts/postinst.sh",
+      "--pkg-install=/opt/zimbra/*"
    );
-   if ( $pkg_info->{dir_list} )
-   {
-      foreach my $expr ( @{ $pkg_info->{dir_list} } )
-      {
-         push( @cmd, "--pkg-installs=$expr" );
-      }
-   }
-   if ( $pkg_info->{file_list} )
```

Comment:
Removing this is risky. 
With this packages get the ownership of /opt/zimbra/*  it will make upgrade and removal of the package trouble some. Especially working with rhel. 

We should mention individual files and the directories that can only be used by the package.

Reply:
Fixed